WebInstall/explainer.md

```diff
@@ -300,6 +300,24 @@ If the user were to deny the permission to install for the origin, they could br
 
 To protect the users privacy, the API does not create any new error names for the `DOMException`, instead it uses 2 common existing names: `AbortError` and `DataError`. This makes it harder for the developer to know if an installation failed because of a mismatch in id values, a wrong manifest file URL or if there is no id defined in the manifest.
 
+**The promise will reject with an `AbortError` if:**
```

Comment:
also: The install permission is required but hasn't been granted.

Reply:
added this